src/main/java/co/rsk/federate/config/PowpegNodeSystemProperties.java

```diff
@@ -0,0 +1,137 @@
+package co.rsk.federate.config;
+
+import static co.rsk.federate.config.PowpegNodeConfigParameter.*;
+
+import co.rsk.config.ConfigLoader;
+import co.rsk.config.RskSystemProperties;
+import co.rsk.federate.signing.config.SignerConfig;
+import com.typesafe.config.Config;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PowpegNodeSystemProperties extends RskSystemProperties {
+
+  public static final String REGTEST = "regtest";
+
+  public PowpegNodeSystemProperties(ConfigLoader loader) {
+    super(loader);
+  }
+
+  public boolean isFederatorEnabled() {
+    return getBoolean(
+        FEDERATOR_ENABLED.getPath(),
+        FEDERATOR_ENABLED.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean isPegoutEnabled() {
+    return getBoolean(
+        PEGOUT_ENABLED.getPath(),
+        PEGOUT_ENABLED.getDefaultValue(Boolean::parseBoolean));
+  }
+
+  public boolean isUpdateBridgeTimerEnabled() {
+    return !netName().equals(REGTEST) ||
```

Comment:
So it's true for any network != regtest independently of the value provided in the config. Is that expected? Is that how it was before?

Reply:
Yes, this is how was before:

```java
    public boolean isUpdateBridgeTimerEnabled() {
        return !this.netName().equals("regtest") ||
            (!configFromFiles.hasPath("federator.updateBridgeTimerEnabled") ||
                configFromFiles.getBoolean("federator.updateBridgeTimerEnabled"));
    }
```